airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresDestination.java

```diff
@@ -215,7 +216,7 @@ private static void writeStreamsWithNRecords(int minRecords,
                                                                                             int batchSize,
                                                                                             CloseableQueue<byte[]> writeBuffer,
                                                                                             String tmpTableName) {
-      InsertValuesStep3<Record, String, JSONB, OffsetDateTime> step = ctx.insertInto(table(tmpTableName), field("ab_id", String.class),
+      InsertValuesStep3<Record, String, JSONB, OffsetDateTime> step = ctx.insertInto(table(name(tmpTableName)), field("ab_id", String.class),
```

Comment:
`name` adds escaping. This is really what we should be doing everywhere if we also want to be able to handle quotes and other character.